### DISCUSSION TITLE
Getting key(s) from this structure

### DISCUSSION THREAD

**Discussion by pick** [nlohmann/json#3306] [Q&A] [answered]
Hi, I have searched around but could not find an example that fixed my issue.

I have a large json file which I am structuring like so,

json items = {
{"100" , "ItemName", 0},
{"200" , "ItemName2", 1},
{"300" , "ItemName3", 2},
.. etc

Here I am trying to get the "ItemName" and the integer after it.

Earlier I was testing this same structure but with only two "items"
{"100" , "ItemName"},
{"200" , "ItemName2"},
{"300" , "ItemName3"},
... etc

I was able to get the ItemName by using this
items.at("100") would return with ItemName

If someone could give an example or if I missed a feature that would be great. Thanks

**Comment by nlohmann** [comment, marked as answer]
See https://json.nlohmann.me/api/basic_json/items/

**Reply by pick** [reply]
Did it a different way but thanks for getting back to me
